packages/react/src/components/Chat/Chat.tsx

```diff
@@ -1,80 +1,126 @@
   /** Accessibility behavior if overridden by the user. */
-  accessibility?: Accessibility
+  accessibility?: Accessibility<ChatBehaviorProps>
 
   /** Shorthand array of the items inside the chat. */
   items?: ShorthandCollection<ChatItemProps>
 }
 
-class Chat extends UIComponent<WithAsProp<ChatProps>, any> {
-  static displayName = 'Chat'
+export type ChatStylesProps = {}
 
-  static className = 'ui-chat'
+const Chat: React.FC<WithAsProp<ChatProps>> &
+  FluentComponentStaticProps<ChatProps> & {
+    slotClassNames: ChatSlotClassNames
+    Item: typeof ChatItem
+    Message: typeof ChatMessage
+  } = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext)
+  const { setStart, setEnd } = useTelemetry(Chat.displayName, context.telemetry)
+  setStart()
 
-  static slotClassNames: ChatSlotClassNames = {
-    item: `${Chat.className}__item`,
-  }
+  const { accessibility, children, className, design, items, styles, variables } = props
 
-  static propTypes = {
-    ...commonPropTypes.createCommon({
-      content: false,
+  const getA11Props = useAccessibility(accessibility, {
+    debugName: Chat.displayName,
+    rtl: context.rtl,
+  })
+  const { classes } = useStyles<ChatStylesProps>(Chat.displayName, {
+    className: Chat.className,
+    mapPropsToInlineStyles: () => ({
+      className,
+      design,
+      styles,
+      variables,
     }),
-    items: PropTypes.arrayOf(customPropTypes.itemShorthand),
-  }
-
-  static defaultProps = {
-    accessibility: chatBehavior,
-    as: 'ul',
-  }
-
-  static Item = ChatItem
-  static Message = ChatMessage
-
-  renderComponent({ ElementType, classes, accessibility, unhandledProps }) {
-    const { children, items } = this.props
-
-    return (
-      <ElementType
-        className={classes.root}
-        {...accessibility.attributes.root}
-        {...rtlTextContainer.getAttributes({ forElements: [children] })}
-        {...unhandledProps}
-        {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.root, unhandledProps)}
-      >
-        {childrenExist(children)
-          ? children
-          : _.map(items, item =>
-              ChatItem.create(item, {
-                defaultProps: () => ({ className: Chat.slotClassNames.item }),
-              }),
-            )}
-      </ElementType>
-    )
-  }
+    rtl: context.rtl,
+  })
+
+  const ElementType = getElementType(props)
+  const unhandledProps = getUnhandledProps(Chat.handledProps, props)
+
+  const element = getA11Props.unstable_wrapWithFocusZone(
+    <ElementType
+      {...getA11Props('root', {
+        className: classes.root,
+        ...rtlTextContainer.getAttributes({ forElements: [children] }),
+        ...unhandledProps,
+      })}
+    >
+      {childrenExist(children)
+        ? children
+        : _.map(items, item =>
+            ChatItem.create(item, {
+              defaultProps: () => ({ className: Chat.slotClassNames.item }),
```

Comment:
Shouldn't we by convention send here `resolvedStyles.item`? I hope not, but still, thought I'd ask

Reply:
Discussed offline: let's avoid changes there and keep previous behavior.